### DISCUSSION TITLE
WAR is disabled instantly

### DISCUSSION THREAD

**Discussion by usauerbrey** [edterbak/HeishaMoNR#152] [General]
When I activate WAR, it is active for a second and then automatically disabled. The log says:
08/01 17:39:46 - Function - WAR: 0
08/01 17:39:45 - Function - WAR: 1
08/01 17:38:42 - Function - WAR: 0
08/01 17:38:41 - Function - WAR: 1
08/01 17:34:29 - Function - WAR: 0
08/01 17:34:29 - Function WAR: Compensation curve active. Using internal Panasonic WAR function. NodeRed WAR function is now disabled
08/01 17:34:28 - Function - WAR: 1

Why is this and what can I do to use WAR
Thanks Uwe

**Comment by edterbak** [comment]
Hi,

The answer is in your log.
` Compensation curve active. Using internal Panasonic WAR function. NodeRed WAR function is now disabled`

Your heatpump is setup using the compensation curve. You have set 2 outside temperatures and gave 2 corresponding water temperatures. This you have done in the wall-mounted controller.


The node-red function WAR, is actually exacly the same as the Panasonic native 'Compensation Curve'. Only difference is that the panasonic native implementation is fixed to the outside temperature of the unit (or the external outside temp sensor), where the node red 'WAR-function' is able to use ANY input signal/sensor for this. 
For me the WAR function is nice to use because the sensor on the unit is sometimes catching direct sunlight, causeing the measured temperature to rise 10-15 degrees. Not stable enough in this case. 

But.. You can not use both at the same time. That is futile and illogical. :)

So... 
The reason it switches off, is that the flow detects you are using the Panasonic WAR function. 
You need to choose which solution you want to use. :) both could be fine.

**Comment by edterbak** [comment]
AND.
Please do not forget.. !

IN case you want to keep using the Panasonic Compensation Curve (which is fine), make sure the node red WAR function is set up and containing the same 4 temperatures in the wall-mounted controller. 

Like this:
![image](https://github.com/edterbak/NodeRed_Heishamon_control/assets/3155621/e30db8bb-e2a0-4410-b356-8156c12e9740)

The top two specifically
14 / 26
-5 / 30

the last two (-6 / 30 ) are less important. Make sure the line looks nice.

**Comment by usauerbrey** [comment]
Hi Ed
Thanks for your detailed information. "But.. You can not use both at the same time. That is futile and illogical. :)"
makes sense, of course.
Besides the sensor problem using the NR-WAR has the advantage, that you can change the curve remotly, which is not possible using the wall-mounted controller ;-))

**Comment by edterbak** [comment]
You can change the Compensation curve as well remotely. Use the EXPORT button in the WAR-function tab. :)  That way you send 4 new temperatures. 
As long as you make sure you only use the top 2 sets of temperatures for that, you should be able to do that remotly too.

Just to note:
The curve as a whole can be shifted up or down from the home page as well. [Shift curve] by -5 till +5 °C. 

The biggest advantage of the NR-WAR function (where I use it for) is that you can use any sensor. 
So you dont 'need' to buy the official panasonic external outside temperature sensor.
You can also use an online source as well, like me "OpenWeatherMap"
